doc/deploy/android.md

```diff
@@ -1,33 +1,129 @@
+instructions will be vastly different. You can ask for help or, since you're probably
+running Linux, just translate them like you're probably used to doing._
 
-1. `npm install`
+### General
+- Did you read the [general deploy info](README.md)?
+- Do you have access to the [Google Play Developer Console](play.google.com/apps/publish)?
+ If not, see Adam.
 
-2. Open `config.xml`. On line 2, increment `android-versionCode` with your appropriately chosen new version number.
+### Android Studio
+We use Android Studio to easily manage the Android SDK tools. If you wish, you can attempt
+to download the tools separately, but these instructions will not help you because I found the
+attempt maddening. Plus, Android Studio will keep track of updates for you, which is nice.
+1. Get [Android Studio](https://developer.android.com/studio/)
+1. Use it to download the SDK platforms and tools: 
+  1. Click menu items Android Studio -> Appearance and Behavior -> System Settings -> Android SDK
+  1. In the main window, click "sdk platforms"
+  1. Click 'edit' to choose an sdk location. Follow the prompts. 
+  The default location is fine--keep note of where that is so you can run the tools 
+  from the command line.  Wait for it to download the components.
+  1. Keep waiting
+  1. Check the boxes next to Android 8.0(Oreo) and any others leading up to the latest--you'll see that that's API 
+  level 26, which we target at the moment. 
+  1. Click on the "tools" tab. Make sure "Android SDK Platform Tools", "Android SDK Tools", and
+  "Android SDK Build Tools" are checked. If you don't have access to an Android phone, make sure
+  the Emulator is also installed. Click "apply"
 
-3. (PVTrAck 2+) `ionic build android --prod --release`
+### Java 1.8
+  1. `$brew update`
+  1. `$brew tap caskroom/versions`
+  1. `brew search java`
+  1. `brew cask install java8`
+  1. If you can figure out why this app requires an old version of Java, please
+  fix it.
 
-    Troubleshooting: if you get an error related to whether this is an Android project, try the following:
+### Gradle
+  1. `$brew install gradle`
+  1. Follow any additional commands to set Gradle up.
 
+### Node
+  1. Install node: `$brew install node`.
+  1. Then use it actually install our packages:`$npm install` (You probably
+  already did this when doing work on the application, but it can't hurt to
+  update if you pulled from master)
+
+## Ready to build?
+  1. Open `config.xml`: On line 2, increment `android-versionCode`
+  with your appropriately chosen new version number.
+  1. You may make a debug build first to try the changes to your application on
+  an emulator or phone: `$ionic cordova build android`.
+  1. If you have an Android: Plug it into the computer with a USB cable and enable USB
+  debugging. Side-load like so, replacing the paths with your own:
+ `~/Library/Android/sdk/platform-tools/adb install /Users/my_name/pvta-multiplatform/platforms/android/build/outputs/apk/debug/android-debug.apk
```

Comment:
Better to use placeholders here rather than full paths: `<android_sdk>/platform_tools/adb install platforms/android/build/outputs/apk/debug/android-debug.apk`

Reply:
If I read that I'd be flummoxed by what that meant, but hopefully the people that read and use this have knowledge and minds more like yours and less like mine.